torch_xla/experimental/fori_loop.py

```diff
@@ -73,7 +73,7 @@ def _xla_while_loop(cond_fn,
   for carried_input in carried_inputs:
     device = carried_input.device
     fake_carried_inputs.append(
-        torch.randint(10, carried_input.size(),
+        torch.randint(2, carried_input.size(),
```

Comment:
can we just use `torch.empty` instead? It is a placeholder anyway.

Reply:
I agree with that `torch.empty` is a better choice, I pushed the new changes